deploy.sh

```diff
@@ -1,4 +1,17 @@
 #!/bin/bash
+
+export KRUIZE_UI_IMAGE="docker.io/bharathappali/kruize-ui:central-config"
```

Comment:
This needs to be changed

Reply:
updated to quay.io/kruize/kruize-ui:test